appyx-interactions/sample-test-driver/src/jsMain/kotlin/com/bumble/appyx/interactions/Children.kt

```diff
@@ -0,0 +1,156 @@
+package com.bumble.appyx.interactions
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.LaunchedEffect
+import androidx.compose.runtime.collectAsState
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.key
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.rememberCoroutineScope
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.composed
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.draw.clipToBounds
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.layout.boundsInRoot
+import androidx.compose.ui.layout.onPlaced
+import androidx.compose.ui.platform.LocalDensity
+import androidx.compose.ui.semantics.contentDescription
+import androidx.compose.ui.semantics.semantics
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import com.bumble.appyx.interactions.core.model.BaseInteractionModel
+import com.bumble.appyx.interactions.core.ui.context.TransitionBounds
+import com.bumble.appyx.interactions.core.ui.context.UiContext
+import com.bumble.appyx.interactions.core.ui.output.ElementUiModel
+import com.bumble.appyx.transitionmodel.testdrive.ui.md_amber_500
+import com.bumble.appyx.transitionmodel.testdrive.ui.md_blue_500
+import com.bumble.appyx.transitionmodel.testdrive.ui.md_blue_grey_500
+import com.bumble.appyx.transitionmodel.testdrive.ui.md_cyan_500
+import com.bumble.appyx.transitionmodel.testdrive.ui.md_grey_500
+import com.bumble.appyx.transitionmodel.testdrive.ui.md_indigo_500
+import com.bumble.appyx.transitionmodel.testdrive.ui.md_light_blue_500
+import com.bumble.appyx.transitionmodel.testdrive.ui.md_light_green_500
+import com.bumble.appyx.transitionmodel.testdrive.ui.md_lime_500
+import com.bumble.appyx.transitionmodel.testdrive.ui.md_pink_500
+import com.bumble.appyx.transitionmodel.testdrive.ui.md_teal_500
+
+val manatee = Color(0xFF8D99AE)
+val silver_sand = Color(0xFFBDC6D1)
+val sizzling_red = Color(0xFFF05D5E)
+val atomic_tangerine = Color(0xFFF0965D)
+
+val colors = listOf(
+    manatee,
+    sizzling_red,
+    atomic_tangerine,
+    silver_sand,
+    md_pink_500,
+    md_indigo_500,
+    md_blue_500,
+    md_light_blue_500,
+    md_cyan_500,
+    md_teal_500,
+    md_light_green_500,
+    md_lime_500,
+    md_amber_500,
+    md_grey_500,
+    md_blue_grey_500
+)
```

Comment:
I would prefer these extracted – they belong to the sample using `Children`, not to `Children` itself.

Reply:
That is to say, we should have a single central `Children` composable (without these specific UI bits) rather than a copy-pasted one per platform.